modules/cudf/test/cudf-column-tests.ts

```diff
@@ -348,3 +350,34 @@ describe('Column.setNullMask', () => {
     validateSetNullMask(makeTestColumn(4), 4, 4, new Uint8Buffer(new DeviceMemory(8)).fill(0));
   });
 });
+
+describe('Column.read_text', () => {
+  test('can read a json file', async () => {
+    const rows = [
+      {a: 0, b: 1.0, c: '2'},
+      {a: 1, b: 2.0, c: '3'},
+      {a: 2, b: 3.0, c: '4'},
+    ];
+    const outputString = JSON.stringify(rows)
+    const path         = Path.join(readTextTmpDir, 'simple.txt');
+    await promises.writeFile(path, outputString);
+    const text = Column.read_text(path);
```

Comment:
It looks like in the C++ implementation, this function expects to be passed the delimiter.